ffs/scheduler/internal/cistore/cistore.go

```diff
@@ -45,8 +45,8 @@ func (s *Store) Get(c cid.Cid) (ffs.CidInfo, error) {
 	return ci, nil
 }
 
-// Put saves a new storing state for a Cid.
-func (s *Store) Put(ci ffs.CidInfo) error {
+// Put saves a new storage state for a Cid.
+func (s *Store) Put(ci ffs.StorageInfo) error {
```

Comment:
So we got lucky that the JSON serialization is naming agnostic.

Reply:
Exactly ;)